backend/functions/attorneys/attorneys.function.test.ts

```diff
@@ -1,20 +1,36 @@
 import { UnknownError } from '../lib/common-errors/unknown-error';
-
-const context = require('azure-function-context-mock');
 import httpTrigger from './attorneys.function';
 import * as httpResponseModule from '../lib/adapters/utils/http-response';
 import { AttorneysController } from '../lib/controllers/attorneys/attorneys.controller';
 import { CamsError } from '../lib/common-errors/cams-error';
+import * as ContextCreator from '../lib/adapters/utils/application-context-creator';
```

Comment:
I would like to make these default exports from the modules so we don't have to use the "import * as" syntax.